simulator/risc_v/t/unit_test.cpp

```diff
@@ -599,6 +599,41 @@ TEST_CASE("RISCV RV64 gorc")
     }
 }
 
+TEST_CASE("RISCV RV32 gorci")
+{
+    CHECK( RISCVInstr<uint32>(0x28D6'5593).get_disasm() == "gorci $a1, $a2, 13");
+    CHECK( RISCVInstr<uint32>(0x28F6'5593).get_disasm() == "gorci $a1, $a2, 15");
+    CHECK( RISCVInstr<uint32>(0x29C6'5593).get_disasm() == "gorci $a1, $a2, 28");
+    std::vector<TestData<uint32>> cases = {
+            {0x1111'0000,  0x1, 0x3333'0000},
+            {0x0022'0033,  0x2, 0x00AA'00FF},
+            {0x0C0C'0F0F,  0x5, 0xCCCC'FFFF},
+            {0x8712'4789,  0xC, all_ones<uint32>()},
+            {1,  0xFF, all_ones<uint32>()},
+            {0x1252'0391,  0x3, 0xFFFF'0FFF},
+    };
+    for(std::size_t i = 0; i < cases.size(); i++) {
+        INFO( "Iteration: " << i);
+        cases[i].make_test("gorci");
+    }
```

Comment:
This flow works for register-to-register instructions:

https://github.com/MIPT-ILab/mipt-mips/blob/master/simulator/risc_v/t/riscv_test_wrapper.h#L15

To test register-to-immediate cases, we pass immediate values to "make_test".

https://github.com/MIPT-ILab/mipt-mips/blob/master/simulator/risc_v/t/riscv_test_wrapper.h#L21